examples/ansible.cfg

```diff
@@ -365,6 +365,11 @@
 # disable the Paramiko look for keys function
 #look_for_keys = False
 
+# paramiko will not use GSS-API Authentication unless the gss_auth option
+# is passed. This is required for Kerberos Authentication.
+# Uncomment this line to enable GSS-API Authentication in Paramiko 
+#gss_auth = True
```

Comment:
Also, should not the commented version be the default value ?

Reply:
I followed some of the other config examples that also said "Uncomment this line to enable..."